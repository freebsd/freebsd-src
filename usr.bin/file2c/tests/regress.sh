
echo 1..9

REGRESSION_START($1)

REGRESSION_TEST(`1', `head -c 13 ${SRCDIR}/regress.in | file2c')
REGRESSION_TEST(`2', `head -c 26 ${SRCDIR}/regress.in | file2c PREFIX')
REGRESSION_TEST(`3', `head -c 39 ${SRCDIR}/regress.in | file2c PREFIX SUFFIX')
REGRESSION_TEST(`4', `head -c 52 ${SRCDIR}/regress.in | file2c -x')
REGRESSION_TEST(`5', `head -c 65 ${SRCDIR}/regress.in | file2c -n -1')

REGRESSION_TEST(`6', `head -c  7 ${SRCDIR}/regress.in | file2c -n 1 P S')
REGRESSION_TEST(`7', `head -c 14 ${SRCDIR}/regress.in | file2c -n 2 -x "P S"')
REGRESSION_TEST(`8', `head -c 21 ${SRCDIR}/regress.in | file2c -n 16 P -x S')

REGRESSION_TEST(`9', `file2c "const char data[] = {" ", 0};" <${SRCDIR}/regress.in')

REGRESSION_END()
