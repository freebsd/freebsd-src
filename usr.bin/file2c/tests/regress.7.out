P S
0xaf,0xd4,
0x3b,0xef,
0xe3,0xea,
0xc0,0x16,
0xba,0x71,
0x34,0xcc,
0xc3,0x51
